contracts/PetPark.sol

```diff
@@ -1,7 +1,114 @@
 //SPDX-License-Identifier: Unlicense
 pragma solidity ^0.8.0;
 
-
 contract PetPark {
+    address owner;
+
+    // define struct of animal type, age, and gender
+    struct animalAndDemo {
+        uint animalType;
+        uint age;
+        uint gender;
+    }
+
+    // animals in zoo: animalType and count of that animal in the zoo
```

Comment:
In solidity it is recommended to use [Natspec](https://docs.soliditylang.org/en/v0.5.10/natspec-format.html) format for commenting